packages/flutter_driver/test/src/extension_test.dart

```diff
@@ -410,4 +414,68 @@ void main() {
       );
     });
   });
+
+  group('waitUntilNoPendingChannelMessages', () {
+    FlutterDriverExtension extension;
+    Map<String, dynamic> result;
+
+    setUp(() {
+      extension = FlutterDriverExtension((String arg) async => '', true);
+      result = null;
+    });
+
+    test('returns immediately when there is no pending channel message',
+        () async {
+      FakeAsync().run((FakeAsync async) {
```

Comment:
Calling a variable `async` is just asking for trouble - how about `fakeAsync`?

Reply:
Renamed to fakeAsync.